src/game/engine/smudgetype.h

```diff
@@ -81,11 +76,11 @@ class SmudgeTypeClass : public ObjectTypeClass
     BOOL Is_Crater() { return Crater; }
     BOOL Is_Bib() { return Bib; }
 
-    static SmudgeType From_Name(const char *name);
-    static const char *Name_From(SmudgeType smudge);
     static void Init_Heap();
-    static void One_Time() {}
```

Comment:
Again, no reason to put an empty function in the cpp, except to prevent it being inlined.